scripts/style-check.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+luacheck `find -name  "*.lua"` --codes
```

Comment:
You can write this as, and if you need to ignore files, you can set those in the config file as https://github.com/nvim-treesitter/nvim-treesitter/blob/dedd74ebdbb8a921d95e3c670be9684d9f655c34/.luacheckrc#L5-L7, also you can add the `--codes` option in the config file https://github.com/nvim-treesitter/nvim-treesitter/blob/dedd74ebdbb8a921d95e3c670be9684d9f655c34/.luacheckrc#L3

```suggestion
luacheck .
```